app/api/application/tests/test_forward_request.py

```diff
@@ -0,0 +1,48 @@
+from unittest.mock import MagicMock
+
+import pytest
+
+from app.api.application import forward_request as forward_request_module
+from app.api.application.forward_request import route_and_forward
+from app.api.domain.exception import DownstreamError
+from app.api.domain.forward_request_model import ForwardRequest
+
+
+def test_route_and_forward() -> None:
+    """Tests the route_and_forward function."""
+    # Arrange
+    forward_request = ForwardRequest(
+        application_id="some application",
+        forward_to="https://example.com",
+        patient_nhs_number="some patient",
+        patient_ods_code="some ods code",
+        proxy_nhs_number="some proxy",
+    )
+    mock_client = MagicMock()
+    mock_client.return_value.transform_response.return_value = (
```

Comment:
Are we checking that forward_request() actually gets called on the mock client? Not sure if this is just nitpicking here